packages/kotti-ui/source/kotti-comment/KtComment.vue

```diff
@@ -21,12 +21,14 @@
 			/>
 			<KtCommentInput
 				v-if="userToReply"
-				isInline
+				autofocus
```

Comment:
what's that?

Reply:
`autofocus` tells `CommentTextArea` whether it should focus the input on component mount.
When editing or replying, the input should autofocus; but when the page loads, the input inside `KtCommentInput` shouldn't autofocus.